website/docs/d/cm_volume.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: volume"
+sidebar_current: "docs-rightscale-datasource-volume"
+description: |-
+  Defines a volume datasource to operate against.
+---
+
+# rightscale_volume
+
+Use this data source to get the ID of an existing volume for use in other resources.
+
+## Example Usage 1: Basic configuration of a volume data source
+
+```hcl
+data "rightscale_volume" "a_volume" {
+  cloud_href = "/api/clouds/1"
+
+  filter {
+    name = "my_volume"
+  }
+}
+
+output "volume name" {
+  value = "${data.rightscale_volume.crunis_volume.name}"
+}
+
+output "volume ID" {
+  value = "${data.rightscale_volume.crunis_volume.resource_uid}"
```

Comment:
Can you also use just 'id' here?  (I believe so)

Reply:
`resource_uid` returns, for example `vol-045e33fd18a746c46`
Just tried `id`, and it returns what is called, I think, the resource href `/api/clouds/1/volumes/63NEHKF8B7RP4`, pinging Adam about this, I thought looking at https://github.com/rightscale/terraform-provider-rightscale/blob/master/rightscale/data_source_volume.go#L75-L112 was the way to found out ALL fields that are returned